_posts/2020-01-24-my-favorite-artists.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: "My Favorite Artists And You Can Too!"
+date: 2020-01-24 21:02:00 -0500
+category: blog
+author: Aaron
+author_twitter: shamshirz
+image: /assets/spotify.png
+---
+
+Sharing my recent Spotify 🎧 favorites with Elm and AWS
+![Spotify Logo]({{ "/assets/spotify/spotify.png" | relative_url }}){:class="excerpt-image"}
+
+
+<!--more-->
+I love Spotify. My music taste can often give a better summary of myself than I can. Conveniently, they also have a nice API that can make sharing that with your friends pretty easy too 😏
+
+This year a friend and I shared past playlists as a way to catch up and realized they were a great way to journal. The best way to share that time and experience with each other was to listen to the playlists on our own and at our own pace. This was a surprising, no-pressure way to share myself and I wanted to explore it even more.
+
+## Live Sharing
+
+How can I share a snapshot of myself, without it becoming dated, and without it being an overwhelming task? So far, I had the core data about me - Spotify - and the idea that I wanted it to stay current. There is a very suitable API for the task that will give a list of your most listened to artists over a time range. I found this to be a great starting point. I broke down my wild ideas into a project scoped for a reasonable amount of time and ended up with a serverless API to fetch my top Spotify artists plus a UI widget for the blog to display it 👇. [See how on our Github](https://github.com/SylverStudios/spotify-highlights)
+
+{% include spotifyArtists.html %}
```

Comment:
it took just a bit of brainpower to figure out this is the thing, in action (!). maybe a little caption under it and/or stylistic padding around it could present it as the special thing it is - the centerpiece of the article, really.

Reply:
Update incoming bayB! Was underwhelming for sure.

### Improves
* Layout (spacing and centered header)
* Attention grabbing (now with color!)
* Added the github repo as a `<figcaption>` instead of in the paragraph body

<img width="844" alt="Screen Shot 2020-02-01 at 1 49 35 PM" src="https://user-images.githubusercontent.com/8139629/73597409-b1dfc880-44f9-11ea-8f20-3f16fc72bcfa.png">

Reply:
Hah nice! very well done

Reply:
that looks awesome! great improvement